spec/requests/graphql/queries/events/get_single_event_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Types::QueryType do
+  describe 'display events' do
+    it 'can query one event' do
+      @user_1 = create(:user)
+      create_list(:event, 3, user: @user_1)
+
+      post graphql_path, params: { query: query }
+      result = JSON.parse(response.body)
+
+      expect(@user_1.events.count).to eq(3)
+      
+      expect(result["data"]["getEvent"]).to have_key("name")
```

Comment:
A bit unsure on the testing here. Should we be mocking and stubbing this out?

Reply:
I'm not sure ... I am still fuzzy on mocking and stubbing :/.

Reply:
You're asserting that the event has details of name, date, color and their type but you could also assert their values. Ex: `expect(result["data"]["getEvent"]["name"]).to eq(@user_1.events.first.name)`

Reply:
Done, updated the tests to include the values.